src/main/java/org/entando/kubernetes/controller/app/ComponentManagerDeployableContainer.java

```diff
@@ -128,8 +124,8 @@ public int getMemoryLimitMebibytes() {
     }
 
     @Override
-    public DatabaseSchemaCreationResult getDatabaseSchema() {
-        return dbSchemas.get(DEDB);
+    public Optional<DatabaseSchemaConnectionInfo> getDatabaseSchema() {
+        return databaseSchemaConnectionInfo.stream().findFirst();
```

Comment:
which is the reason to use this syntax instead of a simple `databaseSchemaConnectionInfo.get(0)` ?